cmake/findDependencies.cmake

```diff
@@ -36,7 +41,11 @@ endif()
 
 
 if (EXIV2_ENABLE_NLS)
-    find_package(Intl REQUIRED)
+    find_package(Intl)
```

Comment:
I would not change this line. The idea of having those `EXIV2_ENABLE_XXX` variables is that if they are `ON` we want to make sure that we are using the feature. That's why we are using `find_package(XXX)` with the `REQUIRED`keyword. With the change done here, we could end up using the feature or not ... so even if the user specify in the command line `cmake -DEXIV2_ENABLE_NLS=ON`, it could be that he ends up not using that feature.

However, I am fine with the changes above. This will help to solve the issue described in #966. By default, `EXIV2_ENABLE_NLS` is **ON** on UNIX, but in case it cannot be found, the user has the option to turn that CMake option to **OFF**.

Reply:
I'm OK about leaving `REQUIRED` if we change the default for `EXIV2_ENABLE_NLS` to `Off` on Mac (and probably every platform).

The message generated by CMake when it doesn't fine libintl is cryptic.  So on a "clean" MacOS-X, if the user does `$ cmake ..`, they see:
```
CMake Error at /usr/local/share/cmake-3.15/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Failed to find Gettext libintl (missing: Intl_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/local/share/cmake-3.15/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/share/cmake-3.15/Modules/FindIntl.cmake:48 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  cmake/findDependencies.cmake:39 (find_package)
  CMakeLists.txt:61 (include)


-- Configuring incomplete, errors occurred!
See also "/Users/rmills/gnu/github/exiv2/0.27-maintenance/build/CMakeFiles/CMakeOutput.log".
315 rmills@rmillsmm-macosx:~/gnu/github/exiv2/0.27-maintenance/build $ 
```
Not a nice default, eh? 

How is the user to figure out how to install libintl, or realise to use `-DEXIV2_ENABLE_NLS=Off`.

The way things stand we're making it hard to build using defaults on MacOS-X (and probably other platforms).

Reply:
Sure, for me it is fine too to disable `NLS` on all the platforms. IMHO it is something optional which is not core for the exiv2 library.

Reply:
For doing that, you would need to change the line:

```
cmake_dependent_option(EXIV2_ENABLE_NLS "Build native language support (requires gettext)" ON "UNIX" OFF)
```
In the main CMakeLists.txt, to: 

```
option(EXIV2_ENABLE_NLS "Build native language support (requires gettext)" OFF)
```

Reply:
Thanks for those tips.  I've made the change you recommend.